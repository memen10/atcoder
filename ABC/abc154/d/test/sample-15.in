3 3
700 384 43