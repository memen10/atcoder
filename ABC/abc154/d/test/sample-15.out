565.000000000000
